### DISCUSSION TITLE
Text components don't let me specify `align` attribute

### DISCUSSION THREAD

**Discussion by nivzx** [h2oai/wave#2365] [Q&A] [answered]
Even though in the [docs](https://wave.h2o.ai/docs/api/ui#text_xl) it says that there is an arg called `align` for `text_xl`, `text_l`,... components, they throw an error `TypeError: text_xl() got an unexpected keyword argument 'align'` when I try to define it.
Here's how I used it:
```
ui.text_xl('Text goes here', align='center'),
```
What am I missing here?
cc: @mturoci

**Comment by vopani** [comment]
Whats the version of your Wave package?

**Reply by nivzx** [reply]
1.0.0

**Reply by marek-mihok** [reply, marked as answer]
@nivzx it should be implemented by #2336 and working since [v1.3.0](https://github.com/h2oai/wave/releases/tag/v1.3.0)

Docs always refer to the latest version. Could you please update Wave version and check if it working?
